src/tensor_ops.jl

```diff
@@ -59,7 +66,7 @@ Computes the norm of a tensor
 """
 Computes the outer product between two tensors `t1` and `t2`. Can also be called via the infix operator `⊗`.
 """
-@inline otimes{order, dim}(t1::AbstractTensor{order, dim}, t2::AbstractTensor{order, dim}) = otimes(promote(t1, t2)...)
+# @inline otimes{order, dim}(t1::AbstractTensor{order, dim}, t2::AbstractTensor{order, dim}) = otimes(promote(t1, t2)...)
```

Comment:
Or maybe if the type is different this one will be used? Like if one `Tensor` is `Int` and the other `Float`?